### PR TITLE
ticket 15 - GET /api/articles/:article_id/comments

### PR DESCRIPTION
**Pull request by deasar-js**
retrieves comments from database using article_id, responds with array of objects

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -201,14 +201,40 @@ describe("GET /api/users", () => {
...
-        console.log(body);
         expect(body.msg).toBe("Route not found");
       });
   });
+  test("404 Comments not found", () => {
+    return request(app)
+      .get("/api/articles/999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Comments not found");
```
Comment (douglashellowell) [inline __tests__/index.test.js:237]: wouldn't this be "article not found"? 🤔

**Review by douglashellowell — APPROVED** [review]
Excellent work!

Really neat code, great use of async/await

File: __tests__/index.test.js (hunk 1 above)
Comment (deasar-js) [reply __tests__/index.test.js:237]: I suppose it would!

File: __tests__/index.test.js [hunk 2]
```diff
@@ -201,14 +201,40 @@ describe("GET /api/users", () => {
...
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200 - responds with array of objects representing comments from that article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
```
[the remark this answers is not in the document]
Comment (deasar-js) [reply __tests__/index.test.js:212]: thanks will have a read and try using it next time :)

**Pull request merged**